### PR TITLE
Updated Phrasing

### PR DESCRIPTION
**Pull request by brandonfrulla**
Just a little word-smithing in the first paragraph.

### CODE REVIEW

File: docs/S01-fundamentals/M2-consensus/L2-consensus-additional/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h1>Distributed Coordination: Consensus Protocols</h1>
...
+  <p>After learning about cryptography, we know how create a identity, how to make sure no one has messed with a file, and how to capture a user's intent all in a decentralized peer-to-peer way. This all well and good for current interactions or things we are doing <i>now</i> on a network, but how do we agree on things that have happened in the past? That is, as actors in a network- how do we coordinate and agree on the series of events that have led to the current state of the network? How do we know someone really <i>does</i> have digital money they can send to us and are not making it up? How do all the actors in the network then maintain that knowledge in a secure way? This is what we're going to learn in this module.</p>
```
Comment (kwight) [inline docs/S01-fundamentals/M2-consensus/L2-consensus-additional/index.html:18]: Maybe this instead?

```suggestion
  <p>After learning about cryptography, we know how create a identity, how to make sure no one has messed with a file, and how to capture a user's intent all in a decentralized peer-to-peer way. This all well and good for current interactions or things we are doing <i>now</i> on a network, but how do we agree on things that have happened in the past? That is, as actors in a network, how do we coordinate and agree on the series of events that have led to the current state of the network? How do we know someone really <i>does</i> have digital money they can send to us and are not making it up? How do all the actors in the network then maintain that knowledge in a secure way? This is what we're going to learn in this module.</p>
```

**Comment by kwight** [comment]
Also related to #49 .

File: docs/S01-fundamentals/M2-consensus/L2-consensus-additional/index.html (hunk 1 above)
Comment (brandonfrulla) [reply docs/S01-fundamentals/M2-consensus/L2-consensus-additional/index.html:18]: Works for me

**Comment by ConsenSys-Academy** [comment]
Look like there's a conflict, this may have already been resolved? Thank you for catching though and your link @kwight !

**Comment by ConsenSys-Academy** [comment]
Ah, so there are two things in the line that changed, can you pull the latest changes to the repo, merge them locally and push it back to your branch? I will merge it when you do!

**Comment by cooganb** [comment]
Hey! Lmk if you'd like to try and fix this, otherwise I'll close in 24hrs

**Comment by brandonfrulla** [comment]
Sorry, I thought I did! Is there still an issue merging?

**Comment by cooganb** [comment]
@brandonfrulla yes! If you scroll below this message I'm about to post you'll see this message:

Screenshot_20210907-072704

**Comment by cooganb** [comment]
Check out how to resolve conflicts here: https://github.com/ConsenSys-Academy/Blockchain-Developer-Bootcamp/blob/main/CONTRIBUTING.md#steps-to-github-prs

**Comment by brandonfrulla** [comment]
Screen Shot 2021-09-07 at 9 04 49 AM
@cooganb - Thanks for the link, but I'm getting staging not found when fetching?

**Comment by cooganb** [comment]
@brandonfrulla Can you try one more time?

**Comment by cooganb** [comment]
It honestly is probably best to save your changes, and start fresh!

**Comment by cooganb** [comment]
Closing this for the time being but can answer questions in this thread here!

**Pull request closed without merging**